keps/sig-scheduling/2249-pod-affinity-namespace-selector/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Namespace Selector for Pod Affinity
+kep-number: 2249
+authors:
+  - "@ahg-g"
+owning-sig: sig-scheduling
+participating-sigs:
+status: implementable
+creation-date: 2021-01-11
+reviewers:
+  - "@alculquicondor"
+  - "@Huang-Wei"
+  - "@liggitt"
+approvers:
+  - "@Huang-Wei"
+  - "@liggitt"
+prr-approvers:
+  - "@wojtek-t"
+see-also:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.21"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.21"
+  beta: "v1.22"
+  stable: "v1.24"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: PodAffinityNamespaceSelector
```

Comment:
to force graduating the two features together(namespaceSelector and the new scope), I am using a single feature flag.